internal/controllers/utils.go

```diff
@@ -49,6 +50,21 @@ type ipPolicyResolver struct {
 	client client.Reader
 }
 
+func (r *ipPolicyResolver) validateIPPolicyNames(ctx context.Context, namespace string, namesOrIds []string) error {
+	for _, nameOrId := range namesOrIds {
+		if strings.HasPrefix(nameOrId, "ipp_") && len(nameOrId) == 31 { // TODO better validate
```

Comment:
I was concerned about some edge cases here, but I forgot that k8s doesn't actually allow underscores in names, so this is actually seems pretty solid

```
The IPPolicy "ipp_test" is invalid: metadata.name: Invalid value: "ipp_test": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.'
```